homeworks/alina.yaremenko_AlinaYaremenko/2-logic/logic.js

```diff
@@ -0,0 +1,42 @@
+const firstNumber = 505;
+const secondNumber = 404;
+
+const multiply = '*';
+const divide = '/';
+const add = '+';
+const subtract = '-';
+
+const operation = '*';
+
+if (operation === subtract) {
+    const result = firstNumber - secondNumber;
```

Comment:
1) Let's create this variable in the upper scope (before if) and set it as `let`;
2) Assign each result in this `result` variable;
3) Use only one `console.log` with result at the end of the file.

If will reduce console.log uses and simplify your code, will make it easier to maintain.